optimum/intel/openvino/modeling_sentence_transformers.py

```diff
@@ -0,0 +1,180 @@
+import warnings
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from types import MethodType
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+from huggingface_hub.constants import HUGGINGFACE_HUB_CACHE
+from sentence_transformers import SentenceTransformer
+from transformers import AutoTokenizer, PretrainedConfig
+from transformers.file_utils import add_start_docstrings
+
+from ...exporters.openvino import main_export
+from .configuration import OVConfig, OVWeightQuantizationConfig
+from .modeling import MODEL_START_DOCSTRING, OVModel
+
+
+@add_start_docstrings(
+    """
+    OpenVINO Model for feature extraction tasks for Sentence Transformers.
+    """,
+    MODEL_START_DOCSTRING,
+)
+class OVModelForSentenceTransformer(OVModel):
+    export_feature = "feature-extraction"
+
+    def __init__(self, model=None, config=None, orig_model_id_or_path=None, **kwargs):
```

Comment:
if used to load the tokenizer why not load it in _from_pretrained directly :

```suggestion
    def __init__(self, model=None, config=None, tokenizer=None, **kwargs):
```

Reply:
fixed